src/compositor-rdp.c

```diff
@@ -83,6 +83,31 @@
 #define DEFAULT_AXIS_STEP_DISTANCE 10
 #define RDP_MODE_FREQ 60 * 1000
 
+/* The RDP API is truly wonderful: the pixel format definition changed
+ * from BGRA32 to B8G8R8A8, but some versions ship with a definition of
+ * PIXEL_FORMAT_BGRA32 which doesn't actually build. Try really, really,
+ * hard to find one which does. */
+#if FREERDP_VERSION_MAJOR >= 2 && defined(PIXEL_FORMAT_BGRA32) && !defined(RDP_PIXEL_FORMAT_B8G8R8A8)
+#define DEFAULT_PIXEL_FORMAT PIXEL_FORMAT_BGRA32
+#else
+#define DEFAULT_PIXEL_FORMAT RDP_PIXEL_FORMAT_B8G8R8A8
```

Comment:
I wouldn´t care for this, if the definition is missing freerdp is way too old (pre `2.0-rc0`) and you´ll most likely have a lot of troubles later on. Better abort if `PIXEL_FORMAT_BGRA32` is not found.

Reply:
So I'll provide patch, that basically replaces RDP_PIXEL_FORMAT_B8G8R8A8 with PIXEL_FORMAT_BGRA32, right?

Reply:
No, if you encounter that format simply abort as freerdp will not properly work with ogon anyway

Reply:
the oldest version of freerdp working with ogon is the one cloned in this project, that is revision a2e7d7139b4aadd29ebbf25db3b8c2ff81cc2b55

Reply:
that is a version not long before `2.0.0-rc0`